tools/downloader/src/open_model_zoo/model_tools/_configuration.py

```diff
@@ -370,10 +385,17 @@ def load_models(args):
     for config_path in sorted(_common.MODEL_ROOT.glob('**/model.yml')):
         subdirectory = config_path.parent.relative_to(_common.MODEL_ROOT)
 
+        composite_model_config = config_path.parents[1] / 'composite-model.yml'
+        composite_model_name = None
+        if composite_model_config.is_file():
+            check_composite_model_dir(composite_model_config.parent)
```

Comment:
Seems like this will recheck the composite model directory for every component of the composite model, which is not ideal. I would suggest that you first find all `composite-model.yml` files, validate them, remember where they were, and then, for each model, determine whether it belongs to one of the composite models.

Reply:
Done